src/tests/system/tests/test_ldap.py

```diff
@@ -14,6 +14,16 @@
 from sssd_test_framework.topology import KnownTopology
 
 
+def clean_restart_sssd(client):
```

Comment:
The framework should have this implemented.

Reply:
https://github.com/SSSD/sssd-test-framework/blob/master/sssd_test_framework/utils/sssd.py#L278